python/understack-flavor-matcher/tests/test_flavor_spec.py

```diff
@@ -316,3 +319,20 @@ def test_large_flavor_memory_slightly_less_disk_exact(flavors):
     )
     # Should not match because memory is slightly less than required
     assert all(flavor.score_machine(machine) == 0 for flavor in flavors)
+
+
+def test_memory_gib(valid_yaml):
+    flv = FlavorSpec.from_yaml(valid_yaml)
+    assert flv.memory_mib == 7963648
+
+
+def test_baremetal_nova_resource_class(valid_yaml):
+    flv = FlavorSpec.from_yaml(valid_yaml)
+    assert (
+        flv.baremetal_nova_resource_class == "RESOURCES:CUSTOM_BAREMETAL_GP2ULTRAMEDIUM"
```

Comment:
Same thing as my other comment, as far as I know, `resources:` needs to be lowercased, but the rest of it uppercased.

Reply:
Fixed